sdk/resourcemanager/azure-resourcemanager-keyvault/src/test/java/com/azure/resourcemanager/keyvault/ManagedHsmTests.java

```diff
@@ -43,6 +44,7 @@ protected void cleanUpResources() {
      *       that's not purged after deletion.</p>
      * @see <a href="https://learn.microsoft.com/en-us/azure/key-vault/managed-hsm/soft-delete-overview">soft-delete-overview</a>
      */
+    @Disabled("Disabled for test-proxy migration. Please enable it again, next time api-version update.")
```

Comment:
Disabled till next api-version.

Don't want to cost 100+ dollars for migration.